packages/strapi-plugin-content-manager/controllers/relations.js

```diff
@@ -31,6 +32,11 @@ module.exports = {
       return ctx.notFound('target.notFound');
     }
 
+    if (idsToOmit && Array.isArray(idsToOmit)) {
+      query._where = query._where || {};
+      query._where.id_nin = concat(query._where.id_nin || [], idsToOmit.map(String));
```

Comment:
do we really need th cast to string here ?

Reply:
not anymore